### ISSUE TITLE
cannot change the password (again)

### ISSUE THREAD

**Issue by dsnalzami** [iot-salzburg/gpu-jupyter#56]
hello, maybe I am not clear gave the description in previous thread: https://github.com/iot-salzburg/gpu-jupyter/issues/54

I used your image to the students in the university lab. after I pulls the image, I put the password from the student. the method is :

nano jupyter_notebook_config.py

paste the password to c.NotebookApp.password = u'student salted password'.

then I restarted the service. 

I always used the above method for your previous image (last month)

but this latest image, I cannot replicate the methods. 

if I using your proposed: "Another solution would be to create a new Dockerfile that pulls cschranz/gpu-jupyter:tag" it would be burden due some student like to change the password often and I used portainer so the representative only use portainer to manage the docker and image (last time we gave them the terminal access, it was catastrophe)

**Comment by ChristophSchranz** [comment]
The reason for this is most likely that we switched from Jupyterlab 2 to version 3. This leads to a different naming of the fields.
The class name `NotebookApp` has changed to `ServerApp`. I hope a simple renaming works for you!

By the way, I've made it easier today to set a password in f6bb51d10b7495dcc409d70ceb698bdfaff7b1b0. Just run, e.g.,
`bash generate-Dockerfile.sh --password your_password --python-only`
